### PR TITLE
add an option to the scheduling exercise

### LINKED ISSUE #1404: clarify that signups can be completed later
## 4. Visual noise and Task Switching (Etherpads)

I found the Etherpad useful during the lesson exercises, when participants were populating content into the page shared between the group. However, the instructor checkout process requires signing up for a community discussion and a teaching demo using the pads at https://pad.carpentries.org/community-discussions and https://pad.carpentries.org/teaching-demos. 

The information density of these two pages can be stressful when just five minutes is allocated for signing up within the training schedule. Being presented with this page (with no indexes or hyperlinks to find specific sections within it!) is a high amount of visual noise and it is difficult to navigate, especially when attention is also split with the Zoom call. Philipp handled this really well and clarified that I could come back to these later, but this could be emphasised within the teaching material, or perhaps a more streamlined way could be found to track sign ups to these sessions.

This could also be due to difficulties task switching: changing from a lecture-based learning approach to having to interpret large amounts of data.

_Originally posted by @sjmf in https://github.com/carpentries/instructor-training/issues/1395#issuecomment-1163278573_

add an option in the [related exercise](https://carpentries.github.io/instructor-training/14-checkout/index.html#schedule-a-discussion-or-demo) to save the links to sign up/ add them to a todo list.

### PR DESCRIPTION
**Pull request by brownsarahm**
closes #1404 per the great advice from #1395

### CODE REVIEW

**Comment by brownsarahm** [comment]
reverted to draft based on [advice](https://github.com/carpentries/instructor-training/issues/1404#issuecomment-1164001848) from @fiveop to provide context here for trainers.  I would like help phrasing that.

**Comment by brownsarahm** [comment]
Yes, that is the right section @brynnelliott I would also appreciate advice on how/if we should put add context for why this option is here to help trainers pass on that context while teaching this section. 

The linked issue #1395 is where we got the idea, so I want to give trainers the right amount of context to help people, but I also don't want to do things that draw unnecessary attention to how this could be an accommodation.

File: _episodes/14-checkout.md [hunk 1]
```diff
@@ -50,8 +50,10 @@ Extensions may be granted for any reason up to 1 year from your training date.
 > If you would prefer to do your teaching demonstration before your discussion,
 > visit the demo Etherpad and sign up there: <https://pad.carpentries.org/teaching-demos>
 > This demo [rubric]({{ page.root }}/demos_rubric/) is provided as a guide for Trainers evaluating potential new Instructors during the teaching demonstration.
->
->
+> 
+> If these pages are hard to navigate or you cannot find a suitable time, save these links where 
+> you can find them again and make a note to schedule these steps of your checkout process.
```
Comment (brownsarahm) [inline _episodes/14-checkout.md:55]:
```suggestion
> you can find them again and make a note to schedule these steps for your checkout process. 
```

**Comment by brynnelliott** [comment]
@brownsarahm I think what you wrote gives enough information, this way if someone doesn't want to disclose other details, then they don't have too.

**Pull request merged**